### PR TITLE
fix(plugins): fixed `stern` repo link and updated version

### LINKED ISSUE #3347: wercker/stern repo no longer exists causing a 404 error on util fetch-tools --all
### Current Behavior

Relevant code block:

https://github.com/garden-io/garden/blob/7e86f3eaeec92c0801d9c61dd8e1cd867f1b4d51/core/src/plugins/kubernetes/logs.ts#L522

There is a fork of the project here: https://github.com/stern/stern

### Your environment

* **OS:** 
* **How I'm running Kubernetes:**  

`garden version`

### PR DESCRIPTION
**Pull request by vvagaytsev**
**What this PR does / why we need it**:
* Used new repo link for `stern` tool
* Updated `stern` to the latest release version

**Which issue(s) this PR fixes**:

Fixes #3347

**Special notes for your reviewer**:

### CODE REVIEW

**Review by Orzelius — APPROVED** [review]

File: core/src/plugins/kubernetes/logs.ts [hunk 1]
```diff
@@ -519,30 +519,31 @@ export const makeServiceLogEntry: (serviceName: string) => PodLogEntryConverter<
 }
 
 // DEPRECATED: Remove stern in v0.13
+// This version has no Darwin ARM support yet. If you add a later release, please add the "arm64" architecture.
```
Comment (stefreak) [inline core/src/plugins/kubernetes/logs.ts:522]: @vvagaytsev stern 1.22.0 has [stern_1.22.0_darwin_arm64.tar.gz](https://github.com/stern/stern/releases/download/v1.22.0/stern_1.22.0_darwin_arm64.tar.gz)

so we should also add that

Comment (vvagaytsev) [reply core/src/plugins/kubernetes/logs.ts:522]: Hmm... I've missed that somehow. I'll file a separate PR to do that. We'll remove `stern` in 0.13, but it's better to add native ARM binary in 0.12.

Comment (stefreak) [reply core/src/plugins/kubernetes/logs.ts:522]: It's not super critical. Just wanted to point out that the comment is outdated

Comment (stefreak) [reply core/src/plugins/kubernetes/logs.ts:522]: If we remove stern anyway, we can also keep it as it is!

Comment (vvagaytsev) [reply core/src/plugins/kubernetes/logs.ts:522]: @stefreak please review: 3349

**Pull request merged**